From: =?utf-8?q?G=C3=B6ran_Weinholt?= <goran@weinholt.se>
Date: Sun, 23 Sep 2018 23:34:05 +0200
Subject: Disable regeneration of unclean src/machines/.index

---
 src/machines/makeautomachine.sh | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/src/machines/makeautomachine.sh b/src/machines/makeautomachine.sh
index f1603b4..a7d136a 100755
--- a/src/machines/makeautomachine.sh
+++ b/src/machines/makeautomachine.sh
@@ -36,14 +36,6 @@ printf "/*\n *  DO NOT EDIT. AUTOMATICALLY CREATED\n */\n\n" >> automachine.cc
 cat automachine_head.cc >> automachine.cc
 
 printf "3"
-rm -f .index
-for a in *.cc; do
-	B=`grep COMMENT $a`
-	if [ z"$B" != z ]; then
-		printf "$a " >> .index
-		echo "$B"|cut -d : -f 2- >> .index
-	fi
-done
 
 printf "2"
 for a in machine_*.cc; do
